lib/src/context.dart

```diff
@@ -4,7 +4,8 @@
 
 import 'dart:math' as math;
 
-import '../path.dart' as p;
+import 'package:file/file.dart' as fs;
```

Comment:
I left a comment on the original issue - I think this is an SDK problem if we can confirm that `Uri.base` does something unexpected in the zone with an overridden filesystem.

If we _do_ decide that we need to do something in this package we'll need to find a way to do it without depending on `package:file`. This is a _very_ core package in the Dart ecosystem, and maintaining no dependencies is critical.